pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -1,75 +1,180 @@
-        <div class="card-body">
-          <% if (authz_result.mode == 'Exam' || authz_result.mode == 'SEB' || authz_result.password != null ) { %>
-          <p class="lead text-center">
-            Please wait until instructed to start by a proctor
-          </p>
-          <% } %>
 
-          <p class="lead text-center">
-            This is <strong><%= assessment_set.name %> <%= assessment.number %>: <%= assessment.title %></strong> for <strong><%= course.short_name %></strong>
-          </p>
+<head>
+  <%- include('../partials/head'); %>
+</head>
 
-          <% if (authz_result.time_limit_min != undefined) { %>
-          <p class="lead text-center">
-            The time limit for this assessment is
-            <strong>
-              <%= authz_result.time_limit_min %>
-              <%= (authz_result.time_limit_min == 1) ? 'minute' : 'minutes' %>
-            </strong>
-          </p>
-          <% } %>
+<body>
+  <%- include('../partials/navbar', {navPage: ''}); %>
+  <div id="content" class="container">
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white">
+        <%= assessment_set.abbreviation %><%= assessment.number %>: <%= assessment.title %>
+      </div>
 
-          <% if (locals.assessment.require_honor_code) { %>
-          <div class="card card-secondary mb-4 test-class-honor-code">
-            <ul class="list-group list-group-flush">
-              <li class="list-group-item py-2">I certify that I am <%= user.name %> and I am allowed to take this assessment.</li>
-              <li class="list-group-item py-2">I pledge on my honor that I will not give or receive any unauthorized assistance on this assessment and that all work will be my own.</li>
-            </ul>
+      <div class="card-body">
+        <% if (authz_result.mode == 'Exam' || authz_result.mode == 'SEB' || authz_result.password != null ) { %>
+        <p class="lead text-center">
+          Please wait until instructed to start by a proctor
+        </p>
+        <% } %>
 
-            <div class="card-footer text-center border-top-0 py-2">
-              <span class="form-check d-inline">
-                <input type="checkbox" class="form-check-input" id="certify-pledge">
-                <label class="form-check-label font-weight-bold" for="certify-pledge">I certify and pledge the above.</label>
-              </span>
+        <p class="lead text-center">
+          This is <strong><%= assessment_set.name %> <%= assessment.number %>: <%= assessment.title %></strong> for
+          <strong><%= course.short_name %></strong>
+        </p>
+        <% if (authz_result.time_limit_min != undefined) { %>
+        <p class="lead text-center">
+          The time limit for this assessment is
+          <strong>
+            <%= authz_result.time_limit_min %>
+            <%= (authz_result.time_limit_min == 1) ? 'minute' : 'minutes' %>
+          </strong>
+        </p>
+        <% } %>
+        <% if (assessment.group_work) { %>
```

Comment:
All this code looks essentially identical to the copy in `pages/studentAssessmentHomework`. Can we use a partial to avoid duplicating it?

Reply:
Yes, so I created an ejs template at pages/shared/ for the "group section" for both homework and exam. If you could review #3023 first, it would be great.

Reply:
Oh right, I think I remember you telling me that #3023 needed to go in first. I've reviewed that now.